src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -95,17 +95,27 @@ class DepositServlet(app: EasyDepositApiApp)
       } yield Ok(body = s"""{"doi":"$doi"}""", headers = Map(contentTypeJson))
     }.getOrRecoverWithActionResult
   }
+
+  private def fixLicenseUrl(maybeSchemedValue: Option[SchemedValue]) = {
+    def replace(str: String) = {
+      str.replaceAll("dans.knaw.nl.ezproxy[^/]+/", "dans.knaw.nl/")
```

Comment:
Most licenses do not start with dans.knaw.nl, so this will not work for most of them.